repos/system_upgrade/el7toel8/actors/prepareupgradetransaction/libraries/preparetransaction.py

```diff
@@ -282,3 +281,26 @@ def create_disk_image(path):
 def remove_disk_image(path):
     guard_call(['/bin/umount', '-fl', os.path.join(path, 'mounts')])
     shutil.rmtree(path, ignore_errors=True)
+
+
+def yum2dnf_configuration(overlayfs_info):
+    """
+    Secure yum upgradability.
+
+     (workaround)
+
+    :return:
+    """
+    cmds = [
+        ['mkdir', '-p', '/etc/dnf'],
+        ['cp', '-af', '/etc/yum/*', '/etc/dnf/'],
+        ['rm', '-rf', '/etc/yum/pluginconf.d', '/etc/yum/protected.d', '/etc/yum/vars'],
+        ['ln', '-s', '/etc/dnf/plugins/', '/etc/yum/pluginconf.d'],
+        ['ln', '-s', '/etc/dnf/protected.d/', '/etc/yum/protected.d'],
+        ['ln', '-s', '/etc/dnf/vars/', '/etc/yum/vars']
+    ]
+
+    for cmd in cmds:
+        _unused, error = guard_container_call(overlayfs_info, cmd)
+        if error:
+            produce_warning(error, summary='Yum upgradability may be affected.')
```

Comment:
was this discussed somewhere, that these errors should be only reported and upgrade should go on? If so this should at least be reported to the user, not only logged